cellpack/autopack/upy/simularium/simularium_helper.py

```diff
@@ -109,6 +109,16 @@ def __init__(self, master=None, vi=None):
         self.hext = "dae"
         self.max_fiber_length = 0
 
+    @staticmethod
+    def format_rgb_color(color):
+        need_to_divide = False
+        for ele in color:
+            if ele > 1:
+                need_to_divide = True
+        if need_to_divide:
+            color = np.array(color) / 255
+        return matplotlib.colors.to_hex(color)
```

Comment:
This helps to deal with the two different representations of RGB colors (components with 0-255 vs 0.0-1.0) but leaves open a strange corner case if someone intends to have a color between 0-1 relative to a scale of 255, which in this case will become surprisingly white, instead of almost black.

Reply:
Perhaps we make this distinction based on the type of the value (float vs int) instead of the scale of the components?